linux_service/start_moneo_services.sh

```diff
@@ -24,14 +25,16 @@ if lspci | grep -iq NVIDIA ; then
     procs+=("nvidia_exporter")
 fi
 
-if [[ -n $PublisherMethod ]]; then
+if [[ -n $PublisherMethod ]] ; then
```

Comment:
why did this line need a space, but the others below didn't?

Reply:
Good catch, let me address (for clarity the functionality does not change, just a formatting issue)